python/tvm/relay/frontend/pytorch.py

```diff
@@ -2242,6 +2243,29 @@ def embedding(self, inputs, input_types):
 
         return _op.take(weight, indices.astype("int32"), axis=0)
 
+    def embedding_bag(self, inputs, _):
+        assert len(inputs) == 9, "embedding_bag needs 9 arguments"
+        (
+            weights,
+            indices,
+            offsets_1d,
+            scale_grad_by_freq,
+            mode,
+            sparse,
+            per_sample_weights,
+            include_last_offset,
```

Comment:
is it possible that we can support `sparse`, `per_sample_weights` and `include_last_offset` as well? I think at least for model `dlrm`, `sparce` is needed. If we can't really figure out a way to support `scale_grad_by_freq`, for now my take is that we can explicitly put a check here and fail the compilation if this argument is passed from pytorch.

Reply:
And after we add these arguments with tests, this PR should be in pretty good shape.

Reply:
Per offline discussion, we think that we can leave `scale_grad_by_freq` and `sparse`. It would be great if we can ignore those arguments here in the python end, which makes the logic more clear.